common/reportMetrics.py

```diff
@@ -532,10 +539,13 @@ def replay_sar(metric_file_path, command):
         host = header[2][1:-1]
         # assume DD/MM/YYYY
         date = header[3]
-        format = '%m/%d/%Y %I:%M:%S %p'
+        ts_format = '%m/%d/%Y %I:%M:%S %p'
         metric_file.readline()
         metrics = metric_file.readline()
-        field_names = metrics.split()
+        field_names = metrics.replace('/', 'PER').split()
+        while 'RESTART' in field_names or len(field_names) == 0:
+            metrics = metric_file.readline()
+            field_names = metrics.replace('/', 'PER').split()
```

Comment:
we should not be replacing / with PER, just drop. This will bring data in showing like this "tps" instead of "tp/s". This should also only take place for metrics that have a /s, not all /